docs/release-notes/currentRelease.md

```diff
@@ -1,40 +1,37 @@
 ---
-title: DevOps Docker Builds, Version 2403 (Mar 29 2024)
+title: DevOps Docker Builds, Version 2404 (May 1 2024)
 ---
 
-# Version 2403 Release Notes
+# Version 2404 Release Notes
 
 !!! note "Product release notes"
 For information about product changes, refer to the release notes that can be found on each
 product's [download page](https://www.pingidentity.com/en/resources/downloads.html).
 
-## DevOps Docker Builds, Version 2403 (Mar 29 2024)
+## DevOps Docker Builds, Version 2404 (May 1 2024)
 
 ### New Product Releases
 
+- PingAccess 8.0.0 → 8.0.2
 - PingAccess 7.3.2 → 7.3.3
```

Comment:
Is this line supposed to still be here?

Reply:
It should yes

Reply:
Actually that's weird... the ticket that tracked this work was marked for 2404, but it was already part of the 2403 release. Let me remove this.